src/components/home.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import JSONTree from 'react-json-tree';
+
+import DivJSON from './div_json';
+
+import { createAuth, getTokens, renewIdToken, decodeIdToken, refreshSession, closeSession, signOut } from '../actions/index';
+
+class Home extends Component {
+  componentWillMount() {
+    //  create auth that goes in props
+    this.props.createAuth();
```

Comment:
There's a good section in the redux docs that describe tying together with react:
http://redux.js.org/docs/basics/UsageWithReact.html

It seems like here we want to create a "container component" to handle talking to the store - i.e. using connect and Provider to get access to the store, rather than passing it down through props.